docs/TuxActions.md

```diff
@@ -27,34 +28,38 @@
       </li>
     </ol>
   </li>
-  <li><a href="#TuxActions-Complete-Example">TuxActions Complete Example</a>
+  <li><a href="#Actions-Usage-Guidelines">Actions Usage Guidelines</a></li>
+  <li><a href="#TuxActions-Complete-Example">TuxActions Complete Example</a></li>
 </ol>
 
 ## <a id="Premise"></a>Premise [#](#Premise)
 >Tux Actions is a module providing the interface for creating, listening to, and dispatching actions as per the Flux architecture within Tux.
 
 ## <a id="Implementation"></a>Implementation [#](#Implementation)
-### <a id="Requiring-Actions"></a>Requiring Actions [#](#Requiring-Actions)
+### <a id="Requiring-Actions"></a>1) Requiring Actions [#](#Requiring-Actions)
 Tux uses Facebook's [Flux](https://facebook.github.io/flux/) architecture. However Tux abstracts the interactions with the Dispatcher away via an Actions interface. This interface is exposed via:
 
 ```
     var Actions = require('tux/Actions');
 ```
 
-### <a id="Creating-Action-Categories"></a>Creating Action Categories [#](#Creating-Action-Categories)
-The `Actions` object allows the user to create categories of actions via [Actions.createActionCategory](#createActionCategory). Think of categories as the nouns upon which your action verbs act. For example `messages`, `rooms`, `todos`, `users`. You can then assign a list of verbs to that category, for example `create`, `read`, `update`, and `destroy`. Lastly, you can optionally define a source for the actions, which can be helpful metadata for your store callbacks. Lets take a look at an example: [#](#create-action-category-example)
+### <a id="The-Actions-Dispatcher"></a>2) The Actions Dispatcher [#](#The-Actions-Dispatcher)
+In `Tux` there is no direct interaction with the `Dispatcher` as there is in a normal `Flux` application.  Instead the `Actions` object provides all functionality related to the Dispatcher (both triggering and registering listeners for events).  The `Actions` object simply uses the `Dispatcher` on the backend to handle these operations.  The `Actions` object provides tools for registering callbacks with its dispatcher and creating `actionCategories`, which are responsible for dispatching events. `actionCategories` handle the logic associated with interacting with some category of data in the app.  For example, you would have an `actionCategory` for 'messages', another one for 'todos', another one for 'users', etc.  The `actionCategory` is described in full below.
```

Comment:
Looks slightly strange to have a sentence start with `actionCategories`, as it doesn't start with a capital letter. Recommend modifying that sentence.